crates/macros/tests/debug_job/fail/argument_not_extractor.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0277]: the trait bound `bool: FromJobCall<(), blueprint_core::extract::private::ViaParts>` is not satisfied
+error[E0277]: the trait bound `bool: FromJobCall<(), blueprint_sdk::blueprint_core::extract::private::ViaParts>` is not satisfied
```

Comment:
can you look into why this happens? this isn't a valid path. is it just taking the current crate name for some reason?

not super important, though. it's a path to a private type anyway.

Reply:
this happens because we wrote the tests anyway. also see how we export everything from the blueprint core into our sdk. I guess we wrap it into a `mod core { pub use blueprint_core::*; }` then we can do `pub use self::core::*;` in the sdk.

Reply:
it'd be fine to have `blueprint_sdk::extract::private::ViaParts`, though. where is it getting the `blueprint_core` from?

Reply:
reverted this change for now. I'd rather the test fail than us accept a broken path